src/main/java/slimeknights/tconstruct/blocks/BlockProperties.java

```diff
@@ -54,8 +54,8 @@
   static Block.Properties PUNJI = builder(Material.PLANTS, NO_TOOL, SoundType.PLANT).hardnessAndResistance(3.0F);
 
   // SMELTERY
-  static Block.Properties SMELTERY = builder(Material.ROCK, NO_TOOL, SoundType.METAL).hardnessAndResistance(3.0F, 20.0F);
-  static Block.Properties SMELTERY_GLASS = builder(Material.ROCK, NO_TOOL, SoundType.METAL).hardnessAndResistance(3.0F, 20.0F).notSolid();
+  static Block.Properties SMELTERY = builder(Material.ROCK, NO_TOOL, SoundType.METAL).hardnessAndResistance(3.0F, 9.0F);
```

Comment:
Why would you change the resistance? Their values are intentional.

Reply:
Reason is in the commit message. 1.15's blocks are apparently 100% impervious (or
close to it) to tnt, whereas 1.12 blocks react to tnt about the same as 1.15's end brick.